llm_on_ray/inference/models/hpu/CodeLlama-7b-hf-hpu.yaml

```diff
@@ -0,0 +1,13 @@
+port: 8000
+name: CodeLlama-7b-hf
+route_prefix: /CodeLlama-7b-hf
+num_replicas: 1
+cpus_per_worker: 8
+hpus_per_worker: 1
+device: hpu
+model_description:
+  model_id_or_path: codellama/CodeLlama-7b-hf
+  tokenizer_name_or_path: codellama/CodeLlama-7b-hf
+  chat_processor: ChatModelGptJ
```

Comment:
This config file is not correct. Please refer to the existing yaml file for CPU.